util/arena.h

```diff
@@ -79,6 +79,12 @@ class Arena : public Allocator {
 
   size_t BlockSize() const override { return kBlockSize; }
 
+  bool IsInInlineBlock() const {
+    assert(blocks_memory_ == sizeof(inline_block_) ||
+           blocks_memory_ >= kBlockSize);
+    return blocks_memory_ == sizeof(inline_block_);
```

Comment:
Will do. I think `blocks_memory_ == sizeof(inline_block_)` is correct as well (if `kBlockSize == sizeof(inline_block_)` and one block is allocated on the heap, `blocks_memory_` will be `2*sizeof(inline_block_)`), but I agree that `blocks_.empty()` would be cleaner.